views/chooseAll-answer.pug

```diff
@@ -0,0 +1,45 @@
+if answerForm
+	.form-group.col.s12
+		each index in [1,2,3,4]
+			.row
+				.input-field.col.s1
+					input(type='checkbox', name='checkButton', value='mcans'+index, id='mcans'+index)
+					label(for='mcans'+index)
+				.input-field.col.s10
+					input.form-control(type='text', name='mcans'+index, placeholder='Enter Answer Here', required)
+				.input-field.col.s1
+					a.btn-floating.btn-tiny.waves-effect.waves-light.red(onclick='$(this).parent().parent().remove()')
+						i.tiny.material-icons close
+
+	#addMoreAnswersButton
+		a.waves-effect.waves-light.btn(onclick="addMCAnswers('checkbox')") add answer
+
+else if studentQuestionForm
+	form.form-question#chooseAll_answerForm(style='text-align: left;')
```

Comment:
we should not have any CSS in the style tag since it's bad practive, please move this to a CSS class on its own

Reply:
done